app/src/main/java/com/plaid/linksample/MainActivityJava.java

```diff
@@ -88,11 +88,10 @@ private void openLink() {
     products.add(PlaidProduct.TRANSACTIONS);
     Plaid.openLink(
         this,
-        new LinkConfiguration.Builder()
-            .clientName("Link demo")
-            .products(products)
-            .publicKey(getString(R.string.plaid_public_key))
-            .build());
+        new LinkTokenConfiguration.Builder()
+          .token(getString(R.string.plaid_link_token))
```

Comment:
Maybe add a helper method called: `getLinkToken()` with the comments 
```
Call your server for a secure one-time token. Dummy implementation here
```
return "link-sandbox-replace-me"

Reply:
Makes sense! Is there a way to share the same implementation between kotlin and java? Or should I just duplicate this `getLinkToken()` in both?